processor/processor.go

```diff
@@ -1552,6 +1552,21 @@ type storeMessage struct {
 	rsourcesStats rsources.StatsCollector
 }
 
+func sendRetryStoreStats(attempt int) {
+	pkgLogger.Warnf("Timeout during store jobs in processor module, attempt %d", attempt)
+	stats.NewTaggedStat("jobsdb_store_timeout", stats.CountType, stats.Tags{"attempt": fmt.Sprint(attempt), "module": "processor"}).Count(1)
```

Comment:
we can now setup P1 alerts against these new measurements:
1. jobsdb_store_timeout
2. jobsdb_update_timeout
3. jobsdb_query_timeout

Reply:
https://github.com/rudderlabs/rudderstack-operator/pull/762